### PR TITLE
Add Ember CSI role and playbook to deply Ember CSI via its operator

### PR DESCRIPTION
**Pull request by kirankt**
**What this PR does / why we need it**:
Add support for deploying Ember CSI using its operator

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by rthallisey** [comment]
@kirankt I don't see the CustomResource created.  You're going to need to create it in the role so that the operator launches the app.

File: roles/ember-csi/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+apb_action: "provision"
+ember_csi_namespace: "ember-csi"
+repo_name: "quay.io/kirankt"
```
Comment (Akrog) [inline roles/ember-csi/defaults/main.yml]: Please use embercsi's repository instead.
embercsi/ember-csi-operator:v0.0.3 is available there.

Comment (kirankt) [reply roles/ember-csi/defaults/main.yml]: Changed. Thanks.

**Comment by rthallisey** [comment]
@kirankt we want to have this merged by tomorrow.  Do you think you'll be able to get the README committed in time? Or should we come up with another plan?

**Comment by rthallisey** [comment]
And by README, I mean a doc explaining how to deploy/configure ember-csi from the CLI using the operator to drive it.

**Pull request closed without merging**